src/routes/app/[id]/+page.server.ts

```diff
@@ -3,51 +3,77 @@ import prisma from '$lib/prisma';
 import type { PageServerLoad } from './$types';
 import transporter from '$lib/mailer';
 import { EMAIL_FROM } from '$env/static/private';
+import type { Options } from 'nodemailer/lib/mailer';
 
 export const load = (async ({ params: { id } }) => {
     const app = await prisma.app.findUnique({
         where: { appId: String(id) },
         include: { listings: true }
     });
+    return { app };
+}) satisfies PageServerLoad;
 
-    console.log('Emailing to Jae to verify link');
+export const actions = {
+    default: async ({ request }: { request: any }) => {
+        try {
+            const formData = await request.formData();
+            const email = formData.get('to');
+            const message = {
+                from: EMAIL_FROM,
+                to: email,
```

Comment:
You have to deal with email being string | null.  What if there is no value returned from `formData.get('to')` ?